sd-validation/json.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const jsonlint = require('jsonlint-mod');
+
+/**
+ * @param {string} input
+ * @returns {{error: {message: string, line: string|null}|null, result: Object|null}}
+ */
+module.exports = function parseJSON(input) {
+  let result;
+  const logError = console.error; // eslint-disable-line no-console
+
+  try {
+    // jsonlint-mod always calls console.error when there's an error
+    // We don't want this behavior, so we stash console.error while it's executing
+    console.error = () => undefined; // eslint-disable-line no-console
```

Comment:
Should no longer be needed 🙂 https://github.com/circlecell/jsonlint-mod/pull/2

Reply:
hurray that's awesome! :D